internal/services/tesla_task_service.go

```diff
@@ -51,6 +51,7 @@ type TeslaCredentialsV2 struct {
 	AccessToken   string    `json:"accessToken"`
 	Expiry        time.Time `json:"expiry"`
 	RefreshToken  string    `json:"refreshToken"`
+	APIVersion    int       `json:"version"`
```

Comment:
I would just call them both `version` or both `apiVersion`. These mismatches are a bit odd to me.